### ISSUE TITLE
Why there is no mm9?

### ISSUE THREAD

**Issue by socialtree-yt** [phoebe460/diffTF#3]

**Comment by socialtree-yt** [comment]
Could you please provide a translationTable_mm9.csv and PWMScan for mm9? I don't know how to get a HOCOID for mm9 and I worry about lacking some TFs and genes in these two files. Could you please add relevant mm9 files as mm10,hg19 and hg38?

**Comment by phoebe460** [comment]
Hi @socialtree-yt, so sorry I am just seeing your issue now... 9 months later... have you tried posting this same issue to https://bitbucket.org/chrarnold/difftf/issues?status=new&status=open? I think the actual authors of this tool would be able to provide you actual files for both the table and PWMScan files related to mm9, mm10, and hg19. When I used diffTF back in 2020 and 2021, I only used hg38 for my data, so I'd be happy to provide those files for you, if you still like? Let me know and I can drop them here!

**Comment by socialtree-yt** [comment]
> Hi @socialtree-yt, so sorry I am just seeing your issue now... 9 months later... have you tried posting this same issue to https://bitbucket.org/chrarnold/difftf/issues?status=new&status=open? I think the actual authors of this tool would be able to provide you actual files for both the table and PWMScan files related to mm9, mm10, and hg19. When I used diffTF back in 2020 and 2021, I only used hg38 for my data, so I'd be happy to provide those files for you, if you still like? Let me know and I can drop them here!

Yes, I'd be happy if you could provide hg38-related files for me. I didn't use this tools actually but I still need it. Thank you very much for your help!!!

**Comment by phoebe460** [comment]
Hi @socialtree-yt , sorry again for my super delayed reply back... I am currently revisiting and using the tool again for my own work at the moment. I am not sure if you're still needing the hg38 files, but I actually use the TFBS motif databases provided by CIS-BP, HOCOMOCO and Swiss Regulon when I test out my samples with the tool. Is there a specific database you're targeting at this time? Let me know and I'd be willing to provide some files for you to use such as the PWM Scan files and the translation table.

**Comment by dmanahan1** [comment]
Hello @phoebe460 , just jumping in here...

I am also trying to figure out how to use diffTF at the moment, and was hoping for an mm19 translation table, i.e. the current build of the mouse genome since 2020. I figured I could just use the HOCOMOCO v13 PWM scan files, but not sure where to get the translation table from. Unless the translation table is something you have?

Thanks.

-David

**Comment by phoebe460** [comment]
Hi @dmanahan1, when I tested out another dataset I was working with using the HOCOMOCO v11 PWM scan files to run diffTF, I used this command to retrieve them in the command line: 
```
wget https://www.embl.de/download/zaugg/diffTF/TFBS/TFBS_hg38_PWMScan_HOCOMOCOv11.tar.gz
```
Then here is attached the translation table that I used for hg38 : [translationTable_hg38.csv](https://github.com/user-attachments/files/23197261/translationTable_hg38.csv). You can go ahead and use that for your work, if that helps. Hope this works out.

**Comment by dmanahan1** [comment]
Hello Phoebe,

Thanks for the quick reply. I am using mouse, not human! (mm19, or GRCm39). Though I assume that means you don't have an updated mg39 translation table available (or the TF beds from PWMScan as required according to the documentation)?

**Comment by phoebe460** [comment]
Sorry for the delayed reply back @dmanahan1 and thanks for your clarification about using the mouse rather than the human TF motif database files. I don't have an updated mm19 or GRCm39 translation table files for you to use, but they do have the older translation table version based on the mm10 mouse genome  when you download their example files using the` git clone https://git.embl.de/grp-zaugg/diffTF.git` command. I have attached that translation table here: 
[translationTable_mm10.csv](https://github.com/user-attachments/files/23291076/translationTable_mm10.csv) for mm10 if you want to use that instead. As for getting the HOCOMOCO v10 PWM scan files for the mouse mm10 genome, you can retrieve them via command line as follows:
`wget https://www.embl.de/download/zaugg/diffTF/TFBS/TFBS_mm10_PWMScan_HOCOMOCOv10.tar.gz`. You can then uncompress the source package using the `tar zxvf TFBS_mm10_PWMScan_HOCOMOCOv10.tar.gz`. Good luck with getting the tool to run with your mouse dataset! It honestly really isn't the easiest tool to get running using the High Performance Cluster (HPC) system based on my experience.
